examples/calculator/DivisionModel.hpp

```diff
@@ -65,7 +76,13 @@ class DivisionModel : public MathOperationDataModel
   }
 
 private:
-
+  void
+  setValidationState(bool isValid, const QString &msg)
```

Comment:
Is this function needed? It just manipulates private member variables and doesn't do anything special....